<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1166</width>
    <height>563</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>993</width>
    <height>429</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SpeechEnhancementDemo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>661</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string>日志</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser_log">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>53</y>
       <width>631</width>
       <height>331</height>
      </rect>
     </property>
     <property name="statusTip">
      <string>日志显示区</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clearlog">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>20</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>200</width>
      <height>130</height>
     </rect>
    </property>
    <property name="title">
     <string>选择设备</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButton_cpu">
       <property name="statusTip">
        <string>使用CPU进行推理(速度慢)</string>
       </property>
       <property name="text">
        <string>CPU</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioButton_gpu">
       <property name="statusTip">
        <string>使用GPU进行推理(速度快)</string>
       </property>
       <property name="text">
        <string>GPU</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>280</y>
      <width>211</width>
      <height>130</height>
     </rect>
    </property>
    <property name="title">
     <string>选择模型</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioButton_tasnet">
       <property name="statusTip">
        <string>TASNET模型(处理速度慢)</string>
       </property>
       <property name="text">
        <string>TASNET</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButton_segan">
       <property name="statusTip">
        <string>SEGAN模型(处理速度快)</string>
       </property>
       <property name="text">
        <string>SEGAN</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_forward">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>430</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>开始推理</string>
    </property>
    <property name="text">
     <string>开始处理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clearall">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>430</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>清空当前处理队列</string>
    </property>
    <property name="text">
     <string>清空队列</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>461</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>系统信息</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>CPU型号:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_cpu">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>321</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>GPU型号:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_gpu">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>321</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>CUDA版本:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>108</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Pytorch版本:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_cuda">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>321</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pytorch">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>321</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_hardware">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>检测环境</string>
     </property>
    </widget>
   </widget>
   <widget class="QToolButton" name="toolButton_readfiles">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>430</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="statusTip">
     <string>将文件加入处理队列</string>
    </property>
    <property name="text">
     <string>读取</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1166</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="statusTip">
    <string>By Nitin 2021.06</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="statusTip">
    <string>退出程序</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
